snapshot/snapshotfs/upload_test.go

```diff
@@ -928,6 +929,7 @@ func TestUpload_VirtualDirectoryWithStreamingFileWithModTime(t *testing.T) {
 			assert.Equal(t, tc.uploadedFiles, atomic.LoadInt32(&man2.Stats.NonCachedFiles))
 			// Cached files don't count towards the total file count.
 			assert.Equal(t, tc.uploadedFiles, atomic.LoadInt32(&man2.Stats.TotalFileCount))
+			require.Equal(t, int64(len(content)), atomic.LoadInt64(&man2.Stats.TotalFileSize))
```

Comment:
Comment on line 922 - `// Second upload may find some cached files depending on timestamps.` had me concerned that the test may not cover cached files but it looks like test case #2 (FixedTime) covers this deterministically i.e. we will always have a cached file here.